prow/hook/events.go

```diff
@@ -33,9 +33,10 @@ func (s *Server) handleReviewEvent(re github.ReviewEvent) {
 		"url":      re.Review.HTMLURL,
 	})
 	l.Infof("Review %s.", re.Action)
-	for p, h := range s.Plugins.ReviewEventHandlers(re.PullRequest.Base.Repo.Owner.Login, re.PullRequest.Base.Repo.Name) {
+	org, repo := re.PullRequest.Base.Repo.Owner.Login, re.PullRequest.Base.Repo.Name
+	for p, h := range s.Plugins.ReviewEventHandlers(org, repo) {
 		go func(p string, h plugins.ReviewEventHandler) {
-			pc := s.Plugins.PluginClient
+			pc := s.pluginClientFor(org, repo)
```

Comment:
This is not thread safe ... but the issue here is that right now we do org/repo resolution here but then the `trigger` plugin needs to do the resolution again -- lots of room for error there if it resolves org/repo incorrectly or differently from here. Hard to have a global read-only plugin config that would be different for each of these calls and resolved here. I can keep the status quo but it doesn't seem right

Reply:
I think it makes sense to revert to current behavior and then try to make sure that the various plugins that need to think about "org/repo" strings use the same code to do so. We can add a simple package that just checks if a given repo is under a given org with simple string matching.